consensus/view_change.go

```diff
@@ -9,112 +9,66 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/harmony-one/bls/ffi/go/bls"
 	msg_pb "github.com/harmony-one/harmony/api/proto/message"
+	"github.com/harmony-one/harmony/core/values"
 	bls_cosi "github.com/harmony-one/harmony/crypto/bls"
 	nodeconfig "github.com/harmony-one/harmony/internal/configs/node"
 	"github.com/harmony-one/harmony/internal/utils"
 	"github.com/harmony-one/harmony/p2p/host"
 )
 
-// PbftPhase  PBFT phases: pre-prepare, prepare and commit
-type PbftPhase int
-
-// Enum for PbftPhase
-const (
-	Announce PbftPhase = iota
-	Prepare
-	Commit
-)
-
-// Mode determines whether a node is in normal or viewchanging mode
-type Mode int
-
-// Enum for node Mode
-const (
-	Normal Mode = iota
-	ViewChanging
-	Syncing
-	Listening
-)
-
-// PbftMode contains mode and viewID of viewchanging
-type PbftMode struct {
-	mode   Mode
+// PBFTMode contains mode and viewID of viewchanging
```

Comment:
BTW, as you are refactoring the naming. Let's conform to use FBFT as it's our consensus's name.